providers/msGraph/src/lib/msgraph.config.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export interface MSGraphConfig {
+  clientId: string;
+  authority: string;
+  secret: string;
```

Comment:
it's better if you will pick these three fields from the `Config` type like `Pick<Config, 'clientId'...> & { ... }` they are optional